src/components/modals/Stake.tsx

```diff
@@ -173,6 +176,47 @@ export const StakeModal: FC<IStakeModalProps> = ({
 								</StakeModalTitleText>
 							</StakeModalTitle>
 							<InnerModal>
+								{stakeState === StakeState.APPROVE ||
+								stakeState === StakeState.APPROVING ||
+								stakeState === StakeState.WRAP ||
+								stakeState === StakeState.WRAPPING ? (
+									<StakeStepsContainer>
+										<StakeStep>
+											<StakeStepTitle>
+												Approve
+											</StakeStepTitle>
+											<StakeStepNumber>1</StakeStepNumber>
+										</StakeStep>
+										<StakeStep>
+											<StakeStepTitle
+												disable={
+													!(
+														stakeState ===
+															StakeState.WRAP ||
+														stakeState ===
+															StakeState.WRAPPING
+													)
+												}
+											>
+												Stake
+											</StakeStepTitle>
+											<StakeStepNumber
+												disable={
+													!(
+														stakeState ===
+															StakeState.WRAP ||
+														stakeState ===
+															StakeState.WRAPPING
+													)
+												}
+											>
+												2
+											</StakeStepNumber>
+										</StakeStep>
+									</StakeStepsContainer>
+								) : (
+									<StakeStepsPlaceholder />
```

Comment:
Do we need this?

Reply:
Yes because of Design we need space there.